apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/SerializationTestData.cs

```diff
@@ -200,5 +201,41 @@ public bool Equals(DictionaryInterfaceContainer other)
                 return Integers.All(pair => other.Integers.TryGetValue(pair.Key, out var otherValue) && pair.Value == otherValue);
             }
         }
+
+        [FirestoreData]
+        public sealed class CustomUser
+        {
+            [FirestoreProperty]
+            public int HighScore { get; set; }
+
+            [FirestoreProperty]
+            public string Name { get; set; }
+
+            [FirestoreProperty]
+            public Email Email { get; set; }
```

Comment:
What if instead of your own Email class (which you can add the Converter annotation to) you were using System.Net.Mail.MailAddress here?  Is there a way to serialize that to/from a String using this API approach?